addons/survey/views/survey_templates.xml

```diff
@@ -134,43 +84,182 @@
+                        <t t-call="survey.survey_button_retake"/>
+                        <div t-if="survey.scoring_type != 'scoring_without_answers'">
+                            If you wish, you can <a t-att-href="'/survey/print/%s?answer_token=%s&amp;review=True' % (survey.access_token, answer.access_token)">review your answers</a>
+                        </div>
+                    </div>
+                    <div class="col-6 text-center" t-if="survey.certification_give_badge and answer.scoring_success">
+                        <img t-att-src="'/web/image/gamification.badge/%s/image_128' % survey.certification_badge_id.id"/>
+                        <div>You received the badge <span class="font-weight-bold" t-esc="survey.certification_badge_id.name"/>!</div>
                     </div>
                 </div>
             </div>
-            <ol t-if="survey.questions_layout == 'page_per_section'" class="breadcrumb mt8 justify-content-end">
-                <t t-set="can_go_back" t-value="survey.users_can_go_back" />
-                <t t-foreach='survey.page_ids' t-as='breadcrumb_page'>
-                    <li t-attf-class="breadcrumb-item #{'active' if breadcrumb_page == page else ''}">
-                        <t t-if="page == breadcrumb_page">
-                            <!-- Users can only go back and not forward -->
-                            <!-- As soon as we reach the current page, set "can_go_back" to False -->
-                            <t t-set="can_go_back" t-value="False" />
-                        </t>
-                        <t t-if="can_go_back">
-                            <a href="#" t-att-data-previous-page-id="breadcrumb_page.id">
-                                <span t-field='breadcrumb_page.title' />
-                            </a>
-                        </t>
-                        <t t-else="">
-                            <span t-field='breadcrumb_page.title' />
-                        </t>
-                    </li>
-                </t>
-            </ol>
+            <div class="container o_survey_result p-4" t-if="graph_data">
+                <div class="tab-content">
+                    <div role="tabpanel" class="tab-pane active survey_graph"
+                         t-att-data-scoring_percentage="answer.scoring_percentage"
+                         t-att-id="'survey_graph_question_%d' % answer.id"
+                         t-att-data-question_id="answer.id"
+                         data-graph-type="doughnut"
+                         t-att-data-graph-data="graph_data">
+                        <canvas id="doughnut_chart"></canvas>
+                        <span class="o_overall_performance"></span>
+                    </div>
+                </div>
+            </div>
+            <div class="oe_structure"/>
         </div>
     </template>
 
+    <!-- Survey: printable page view (all pages) -->
+    <template id="survey_page_print" name="Survey: print page">
```

Comment:
While moving the print template, maybe we could move it to survey_templates_print.xml ?